src/activatable/activatable.ts

```diff
@@ -91,7 +91,7 @@ export class ActivatableCollection<T extends IMinimalActivatable> extends Activa
   }
 
   protected async doDeactivate() {
-    for (const activatable of this.activatables) {
+    for (const activatable of [...this.activatables].reverse()) {
```

Comment:
Can't just loop in reverse order?

Reply:
I was lazy, and this way seemed kinda cool.

Reply:
It has the weird effect of _looking_ more declarative ("reverse!") than imperative ("i--") while at the same time also needlessly allocating a new array and mutating it, which would drive functional purists mad. 😆

Reply:
```suggestion
   let { length: i } = this.activatables;
   while (i-- >= 0) {
     const activatable = this.activatables[i];
   }
```

Reply:
Guess I'm late.

Reply:
🤣

Reply:
They gotta improve that apply suggestion feature.
The button should be still be clickable aaaand once hit, it'd amend my changes to the squashed commit. XD

Reply:
you know there's a bug in that code... :P

Reply:
it should be `while (i--) {`

Reply:
and THAT is why I did it my way. 😛

Reply:
True true, the last pass is "empty".

Reply:
Updated my suggestion just in case you'd like to apply it. :trollface: